Dockerfile-check

```diff
@@ -0,0 +1,15 @@
+ARG TAG=latest
+
+FROM golang:1.7.6
+
+ENV GOPATH=/go
+
+WORKDIR /go/src/github.com/contiv/netplugin/
+ENTRYPOINT ["/code_checks.sh"]
+
+RUN go get github.com/tools/godep github.com/aktau/github-release \
```

Comment:
Please use https://github.com/contiv/netplugin/blob/master/scripts/deps by invoking the right Makefile target.